website/docs/r/avs_private_cloud.html.markdown

```diff
@@ -0,0 +1,134 @@
+  sku_name            = "AV36"
+
+  management_cluster {
+    cluster_size = 3
+  }
+
+  network_block      = "192.168.48.0/22"
+  internet_connected = false
+  nsxt_password      = "QazWsx13$Edc"
+  vcenter_password   = "QazWsx13$Edc"
+  tags = {
+    foo = "bar"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Azure Vmware Solution Private Cloud. Changing this forces a new Azure Vmware Solution Private Cloud to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Azure Vmware Solution Private Cloud should exist. Changing this forces a new Azure Vmware Solution Private Cloud to be created.
+
+* `location` - (Required) The Azure Region where the Azure Vmware Solution Private Cloud should exist. Changing this forces a new Azure Vmware Solution Private Cloud to be created.
+
+* `management_cluster` - (Required) A `management_cluster` block as defined below.
+
+* `network_block` - (Required) The block of addresses which should be unique across virtual network in your subscription as well as on-premise. Changing this forces a new Azure Vmware Solution Private Cloud to be created.
+
+* `sku_name` - (Required) The name of the SKU. Changing this forces a new Azure Vmware Solution Private Cloud to be created. Possible values are "av20", "av36" and "av36t".
+
+* `internet_connected` - (Optional) Is connected to the internet?
+
+* `nsxt_password` - (Optional) The password of the NSX-T Manager. Changing this forces a new Azure Vmware Solution Private Cloud to be created.
+
+* `vcenter_password` - (Optional) The password of the vCenter admin. Changing this forces a new Azure Vmware Solution Private Cloud to be created.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Azure Vmware Solution Private Cloud.
+
+---
+
+A `management_cluster` block supports the following:
+
+* `cluster_size` - (Required) The size of the cluster.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Azure Vmware Solution Private Cloud.
+
+* `circuit` - A `circuit` block as defined below.
+
+* `hcx_cloud_manager_endpoint` - The endpoint for the HCX Cloud Manager.
+
+* `nsxt_manager_endpoint` - The endpoint for the NSX-T Data Center manager.
+
+* `vcsa_endpoint` - The endpoint for Virtual Center Server Appliance.
+
+* `nsxt_certificate_thumbprint` - The thumbprint of the NSX-T Manager SSL certificate.
+
+* `vcenter_certificate_thumbprint` - The thumbprint of the vCenter Server SSL certificate.
+
+* `management_network` - The network used to access vCenter Server and NSX-T Manager.
+
+* `provisioning_network` - The network which isused for virtual machine cold migration, cloning, and snapshot migration.
+
+* `vmotion_network` - The network which is used for live migration of virtual machines.
```

Comment:
What does "network mean" subnet? ip? name? 

maybe we should be a bit more clear in the naming of this property as to what is returned?

Reply:
It returns "provisioningNetwork":"192.168.50.0/25", would we call it "provisioning_subnet"?  "vmotionNetwork":"192.168.49.128/25", "vmotion_subnet" as well?